assets/javascript/app.js

```diff
@@ -0,0 +1,156 @@
+// ID of the Google Spreadsheet
+var spreadsheetID = "1RznpZAVO9Kb2dYlsFCWE4gPrUJkfkKMvMqytoqHAM_s";
+var url = "https://spreadsheets.google.com/feeds/list/" + spreadsheetID + "/od6/public/values?alt=json";
+var questions = [];
+var index = 0;
+
+var timer = {
+	time: 30,
+	running: false,
+	intervalId: 0,
+
+	reset: function(timeSet = 30, location="timer") {
+		timer.time = timeSet;
+		timer.stop();
+		$("#" + location).text(timer.time); 
+	},
+
+	start: function(timeSet = 30, location = "timer") {
+		if (!timer.running) {
+			timer.time = timeSet;
+			timer.intervalId = setInterval(timer.count, 1000, location);
+			timer.running = true;
+		}
+	},
+
+	stop: function() {
+		clearInterval(timer.intervalId);
+		timer.running = false;
+	},
+
+	count: function(loc) {
+		timer.time--;
```

Comment:
```.time``` could be named something else that would be less ambiguous, such as currentTime, or presentTime.